internal/config/testdata/file_cache_config/invalid_max_download_parallelism.yaml

```diff
@@ -0,0 +1,4 @@
+file-cache:
+  cache-file-for-range-read: true
+  download-parallelism-per-file: 100
+  max-download-parallelism: -10
```

Comment:
Let's add a newline at the end: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline